scripts/superdesk/editor2/editor.js

```diff
@@ -734,12 +734,14 @@ angular.module('superdesk.editor2', [
     }])
     .directive('sdTextEditorBlockText', ['editor', 'spellcheck', '$timeout', 'superdesk', '$q', 'gettextCatalog', 'config',
     function (editor, spellcheck, $timeout, superdesk, $q, gettextCatalog, config) {
+        var TOP_OFFSET = 134; // header height
```

Comment:
I will need to get height of three different elements, so I think this is unnecessary use of resources.